src/components/ContactModal.jsx

```diff
@@ -52,7 +52,7 @@ function ContactModal({
               duration: 0.5,
             },
           }}
-          className="flex flex-col justify-center items-center text-black bg-white sm:w-96 rounded-lg p-10 mx-auto relative w-80 font-sans text-sm sm:text-lg"
+          className="flex flex-col  items-center    text-primary bg-black rounded-lg p-10  relative font-sans md:w-96 text-sm sm:text-lg"
```

Comment:
Not sure if this is something important to mention but I see some extra double and triple spaces which is not necessary.
Before `text-primary` there are triple spaces and some other places there are two.